app/src/main/java/com/x8bit/bitwarden/ui/vault/feature/addedit/VaultAddEditScreen.kt

```diff
@@ -235,7 +235,7 @@ fun VaultAddEditScreen(
         },
     )
 
-    if (pendingDeleteCipher) {
+    if (pendingDeleteCipher && state.canDelete) {
```

Comment:
We should not need this, right?

The other check should handle it all

Reply:
You're right!